### PR TITLE
Make changes to support BRT update

### PR DESCRIPTION
**Pull request by rpdome**
For this PR: https://github.com/AzureAD/ad-accounts-for-android/pull/1418

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ vNext
     - [Part 3] Refactor AccountManager Strategy[IPC part 4] test cases for IPC strategies (#1092)
     - [Part 4] test cases for IPC strategies (#1093)
     - [Part 5.5] make changes to support the Broker API work (#1101)
+- Make changes to support BRT update.
```
Comment (AdamBJohnsonx) [inline changelog.txt:17]: What do they do?

File: common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/ContentProviderStrategy.java [hunk 2]
```diff
@@ -135,6 +135,10 @@ public boolean isBrokerContentProviderAvailable(@NonNull final String targetedBr
 
         final List<ProviderInfo> providers = mContext.getPackageManager()
                 .queryContentProviders(null, 0, 0);
+        
+        if (providers == null){
+            return false;
```
Comment (AdamBJohnsonx) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/ContentProviderStrategy.java:140]: Might want a log here.  I feel like we don't want this to happen.

**Review by AdamBJohnsonx — APPROVED** [review]

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 3]
```diff
@@ -1348,6 +1353,11 @@
          */
         public static final String BROKER_API_REMOVE_BROKER_ACCOUNT_PATH = "/brokerApi/removeBrokerAccount";
 
+        /**
+         * URI Path constant for BrokerApi-to-Broker updateBrt request using ContentProvider.
```
Comment (shahzaibj) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1357]:
```suggestion
         * URI Path constant for Broker-to-Broker API updateBrt request using ContentProvider.
```

**Review by shahzaibj — APPROVED** [review]

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java (hunk 3 above)
Comment (rpdome) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1357]: This is intentional. It's the Broker (host) API to the (active) broker (process)

**Pull request merged**